memgpt/connectors/db.py

```diff
@@ -263,18 +265,17 @@ def insert_many(self, passages: List[Passage], show_progress=True):
             temp_dict = {"doc_id": passage.doc_id, "text": passage.text, "passage_id": passage.passage_id, "vector": passage.embedding}
             data.append(temp_dict)
 
-        if self.table:
+        if self.table is not None:
             self.table.add(data)
         else:
             self.table = self.db.create_table(self.table_name, data=data, mode="overwrite")
 
     def query(self, query: str, query_vec: List[float], top_k: int = 10) -> List[Passage]:
         # Assuming query_vec is of same length as embeddings inside table
```

Comment:
@cpacker @sarahwooders This part is a little confusing to me -- the `query()` function never gets called for local storage and lancedb.
I tried to just set a breakpoint for both and run some examples after attaching the data source and it never got called.. I'm not sure if I'm doing something wrong.
To repro
```
/attach
ask a question
```